packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/overview.md

```diff
@@ -1,7 +1,7 @@
 ## About the Authorization Code grant with PKCE
 
-If you are building a native application, then the Authorization Code flow with a Proof Key for Code Exchange (PKCE) is the recommended method for controlling the access between your application and a resource server. The Authorization Code flow with PKCE is similar to the standard [Authorization Code flow](/docs/guides/implement-grant-type/authcode/main/) with an extra step at the beginning and an extra verification at the end.
+The Authorization Code flow with a Proof Key for Code Exchange (PKCE) is the recommended method for controlling the access between your mobile applications and a resource server. The Authorization Code flow with PKCE is similar to the standard [Authorization Code flow](/docs/guides/implement-grant-type/authcode/main/) with an extra step at the beginning and an extra verification at the end.
```

Comment:
Not sure if this applies to only mobile apps. I think it's also used for SPA apps. Please confirm with fed team.

Reply:
confirmed